pkgs/top-level/python-packages.nix

```diff
@@ -126,6 +126,8 @@ let
     mpi = pkgs.openmpi;
   };
 
+  bootstrapped-pip = callPackage ../development/python-modules/bootstrapped-pip { };
```

Comment:
Maybe it helps to ask if we have a use case for having the bootstrap pip available there? If not, we can hide it until we have one.